assets/extensions/tabs.js

```diff
@@ -46,7 +22,8 @@ const Tabs = () => {
 								'aria-current': 'page',
 							} ) }
 						>
-							{ label } <span className="count">(3)</span>
+							{ label }
+							<span className="count">({ count })</span>
```

Comment:
After the break like here, it lost the space between the label and count. Maybe we could add a `padding-left` in the count.